Runtimes/Core/cmake/modules/AvailabilityMacros.cmake

```diff
@@ -1,4 +1,7 @@
-file(STRINGS "${SwiftCore_SWIFTC_SOURCE_DIR}/utils/availability-macros.def" availability_defs)
+configure_file("${SwiftCore_SWIFTC_SOURCE_DIR}/utils/availability-macros.def"
```

Comment:
Why are we copying this file?

Reply:
To ensure we record a dependency on the original. Otherwise, you have to manually re-run cmake after changing availability-macros.def.

Reply:
(That’s why PR testing failed the first time)

Reply:
I don't have a better solution, so okay.